corehq/apps/commtrack/processing.py

```diff
@@ -66,6 +67,21 @@ def _create_model_for_stock_report(domain, stock_report_helper):
         domain=domain,
     )
 
+LedgerValues = namedtuple('LedgerValues', ['balance', 'delta'])
+
+
+def compute_ledger_values(lazy_original_balance, report_type, quantity):
```

Comment:
`lazy_original_balance` is a function rather than a number because it's only used in the case of a transfer, and it may be computationally expensive for the caller to provide; putting it behind a function lets us only do the work if necessary.

A bit of a hack but I think it works for now. On second thought, I think I'm going to add what I just wrote to the docstring here.